adapters/openrtb_util_test.go

```diff
@@ -2,116 +2,100 @@ package adapters
 
 import (
 	"encoding/json"
-	"github.com/mxmCherry/openrtb/v15/openrtb2"
-	"reflect"
+	"errors"
 	"testing"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestExtractAdapterReqBidderParams(t *testing.T) {
-	type args struct {
-		bidRequest *openrtb2.BidRequest
-	}
```

Comment:
Refactors to remove the args object, since it only contains a single argument I don't see the need for a separate structure.